pages/common/comm.md

```diff
@@ -0,0 +1,24 @@
+# comm
+
+> Select or reject lines common to two files. Both files must be sorted and share collation.
```

Comment:
It might be friendlier to beginners to say "must be sorted by the same criteria" rather than using the "collation" jargon. What do you think?